web_transport/sdk/api/owt/quic/web_transport_factory.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef OWT_WEB_TRANSPORT_WEB_TRANSPORT_FACTORY_H_
+#define OWT_WEB_TRANSPORT_WEB_TRANSPORT_FACTORY_H_
+
+#include "owt/quic/export.h"
+#include "owt/quic/web_transport_client_interface.h"
+
+namespace owt {
+namespace quic {
+
+class WebTransportServerInterface;
+class WebTransportClientInterface;
+
+class OWT_EXPORT WebTransportFactory {
+ public:
+  virtual ~WebTransportFactory() = default;
+
+  /// Create a WebTransportFactory.
+  static WebTransportFactory* Create();
+  /// Create a WebTransportFactory for testing. It will not initialize
+  /// AtExitManager since testing tools will initialize one.
+  static WebTransportFactory* CreateForTesting();
+  // `accepted_origins` is removed at this time because ABI compatible issue.
+  // Ownership of returned value is moved to caller. Returns nullptr if creation
```

Comment:
what's the impact of removing this?

Reply:
We don't need it anymore since WebTransport is over HTTP. I'll remove it.